plugins/azure/storageaccounts/storageAccountsTlsVersion.js

```diff
@@ -33,7 +25,7 @@ module.exports = {
         var locations = helpers.locations(settings.govcloud);
 
         var config = {
```

Comment:
can we assign like 
 var tlsVersion = 1.2
 instead of putting it in config object